molecule_vagrant/test/scenarios/molecule/config_update/converge.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
This file is not necessary I think since only the `create` and `destroy` actions are used by the tests.

Reply:
When developing the test I was running the whole test sequence, so i've used it but you're right, it's probably not needed now.